.github/workflows/main.yml

```diff
@@ -68,25 +68,84 @@ jobs:
   website:
     name: build website
     runs-on: ubuntu-latest
+    defaults:
+      # use login shell to make use of .bash_profile /.bashrc
+      run:
+        shell: bash -l {0}
     steps:
+    - name: Cancel Previous Runs
+      uses: styfle/cancel-workflow-action@0.6.0
+      with:
+        access_token: ${{ github.token }}
```

Comment:
For this I would use concurrency, I implemented it a couple of days ago and it cancels immediately on queue rather than requiring the job to actually start:

https://github.com/mne-tools/mne-python/pull/9592/files

Reply:
Can you find a link to the `github.event.number` item? I can't seem to find it. What does it equal and what does it mean? My worry is that if I push a commit to `main` and then push a tag that the tag job would stop the commit job. The website upload job only happens on pushes (are tags considered pushes too?).

Reply:
Oh maybe I should try reading the links right above the concurrency change. :facepalm: 

I added this config in my last commit but it doesn't seem to have done anything (old jobs still running).

Reply:
> I added this config in my last commit but it doesn't seem to have done anything (old jobs still running).

That's because the old jobs on 4a96a9f weren't part of the named concurrency group yet. If you push *two* quick commits (or had pushed one right after 1823718) then it should have cancelled the old one that had the group assigned

Reply:
Ah makes sense. Thanks.